app/views/collection_drafts/forms/fields/_additional_attribute_fields.html.erb

```diff
@@ -22,6 +22,12 @@
     required: true
    ) %>
 
+  <!-- Passing regex for validation of Value and Data Type fields -->
+  <%= hidden_field_tag 'float-regex-for-validation', CollectionDraftsController::FLOAT_REGEX %>
+  <%= hidden_field_tag 'integer-regex-for-validation', CollectionDraftsController::INT_REGEX %>
+  <%= hidden_field_tag 'date-regex-for-validation', CollectionDraftsController::DATE_REGEX %>
+  <%= hidden_field_tag 'time-regex-for-validation', CollectionDraftsController::TIME_REGEX %>
+  <%= hidden_field_tag 'datetime-regex-for-validation', CollectionDraftsController::DATETIME_REGEX %>
```

Comment:
taken from this [article](http://railscasts.com/episodes/324-passing-data-to-javascript)